modules/kargoBidAdapter.js

```diff
@@ -31,35 +31,57 @@ export const spec = {
     const currency = (currencyObj && currencyObj.adServerCurrency) || 'USD';
     const bidIDs = {};
     const bidSizes = {};
+    const impressions = [];
 
     _each(validBidRequests, bid => {
       bidIDs[bid.bidId] = bid.params.placementId;
       bidSizes[bid.bidId] = bid.sizes;
+      impressions.push({
+        id: bid.bidId,
+        tid: bid.transactionId,
+        pid: bid.params.placementId,
+        code: bid.adUnitCode,
+        floor: bid.floor,
+        banner: bid.mediaTypes.banner,
+        video: bid.mediaTypes.video,
+        fpd: { 
+          gpid: bid.ortb2Imp.gpid
+        },
+        bidRequestCount: bid.bidRequestsCount > 0 ? bid.bidRequestsCount : null,
+        bidderRequestCount: bid.bidderRequestsCount > 0 ? bid.bidderRequestsCount : null,
+        bidderWinCount: bid.bidderWinsCount > 0 ? bid.bidderWinsCount : null,
+      }
+
+      ) 
     });
 
     const firstBidRequest = validBidRequests[0];
 
     const tdid = deepAccess(firstBidRequest, 'userId.tdid')
 
     const transformedParams = Object.assign({}, {
```

Comment:
Perhaps rename this to `adReuestParams` or `krakenRequestParams` or similar

Reply:
+1 for `krakenRequestParams` or even just `krakenParams`. `transformedParams` seems a bit vague.

Reply:
Went with krakenParams as it is shorter